pkg/tfvars/openstack/openstack.go

```diff
@@ -38,6 +38,7 @@ type config struct {
 	BootstrapShim                    string   `json:"openstack_bootstrap_shim_ignition,omitempty"`
 	ExternalDNS                      []string `json:"openstack_external_dns,omitempty"`
 	MasterServerGroupName            string   `json:"openstack_master_server_group_name,omitempty"`
+	MasterServerGroupPolicy          string   `json:"openstack_master_server_group_policy,omitempty"`
```

Comment:
This should not be omitempty, as it is a required variable that must be supplied to terraform.